api/src/Entity/Traits/AddressTrait.php

```diff
@@ -167,30 +167,30 @@ public function setAddressCountry($addressCountry)
         return $this;
```

Comment:
Should we type-hint the rest of the returns for this file?

Reply:
Going to defer this to the PHPStan ticket to tackle all at once.